### DISCUSSION TITLE
GSoC 2022

### DISCUSSION THREAD

**Discussion by kpetridis24** [networkx/networkx#5406] [General]
Hi, i am very excited about project-1 of GSoC (implementation of vf2++). How many people are you planning to accept for this project? Also, what do you think about implementing other graph-related algorithms? I see that you already have a triangle counting algorithm, but this returns the number of triangles times 3, so we could boost its performance to count them only one time, plus we could add a blocked triangle counting.

**Comment by rossbar** [comment]
> Hi, i am very excited about project-1 of GSoC (implementation of vf2++)

Glad to hear it!

> How many people are you planning to accept for this project?

If this project were selected for GSoC specifically, then the requirement is that only one "student" works on each individual project. 

> Also, what do you think about implementing other graph-related algorithms? I see that you already have a triangle counting algorithm, but this returns the number of triangles times 3, so we could boost its performance to count them only one time, plus we could add a blocked triangle counting.

Performance improvements are always welcome! Please don't hesitate to make a concrete proposal via pull request. Be sure to check out the [contributor's guide](https://networkx.org/documentation/latest/developer/contribute.html) for info on how to get started with the development workflow.

**Comment by kpetridis24** [comment]
@MridulS I have completed my application for the GSoC 2022, but the application platform is not available yet. The timeline however states that applications can be uploaded from 4th April, which is today. Is this an issue? Is it common to have a little delay?

**Reply by z3y50n** [reply]
It is 18:00 UTC so depending on your timezone it should be open later.

**Reply by kpetridis24** [reply]
Ok then, thank you!!!

**Comment by kpetridis24** [comment]
[Konstantinos_Petridis_proposal_NetworkX.pdf](https://github.com/networkx/networkx/files/8418784/Konstantinos_Petridis_proposal_NetworkX.pdf)

This is my completed proposal that i am planning to send to NumFocus. I would appreciate any review from the community. Please provide your honest opinions. All kinds of feedback are more than welcome :)
